### PR TITLE
Editorial Review

### PR DESCRIPTION
**Pull request by SandraGH5**
Adding syntax edits.

### CODE REVIEW

**Review by EngHabu — DISMISSED** [review]

**Comment by EngHabu** [comment]
@SandraGH5 please run the following commands in your terminal to resolve conflicts:
```bash
git checkout master
git pull
git checkout SandraGH5-edits-3
git pull origin master
```

Then open this file `rsts/concepts/basics.rst` and look for `<<<` they will surround lines that are conflicting between your branch and master... you will have to decide what the right merge is... and remove the lines that have `<<<` and `===` and `>>>` etc.

```rst
<<<<<<< SandraGH5-edits-3
         :type: ref
         :text: Workflow & Task Versioning
         :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    A deeper dive into one of Flyte's most important features: versioning of workflows and tasks.
=======
        :type: ref
        :text: Workflow & Task Versioning
        :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    A deeper dive into one of the Flyte's most important features: versioning of workflows and tasks.
>>>>>>> master
```

Then run this:
```bash
git add -A
git merge --continue
git push
```

File: rsts/concepts/basics.rst [hunk 1]
```diff
@@ -13,8 +13,8 @@ Concepts
        :text: Tasks
        :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    A **Task** is any independent unit of processing. They can be pure functions or functions with side-effects.
-    Each definition of a task also has associated configuration and requirements specifications.
+    A **Task** is any independent unit of processing. Tasks can be pure functions or functions with side-effects.
+    Each definition of a task also has associated configurations and requirements specifications.
```
Comment (cosmicBboy) [inline rsts/concepts/basics.rst:17]:
```suggestion
    Each task is also associated configurations and requirements.
```

File: rsts/concepts/basics.rst [hunk 2]
```diff
@@ -96,10 +96,10 @@ Concepts
 
     .. link-button:: divedeep-data-management
         :type: ref
-        :text: How Flyte Manages Data?
+        :text: How Flyte Manages Data
         :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    What is **metadata**? How large amounts of **raw data** is handled? How data flows between tasks?
+    What is **metadata**? How are large amounts of **raw data** handled? How does data flow between tasks?
```
Comment (cosmicBboy) [inline rsts/concepts/basics.rst:102]:
```suggestion
    As a data-aware orchestration tool, Flyte only keeps track of the **metadata** passing through your tasks and workflows, keeping **raw data** within your own data stores.
```

Comment (SandraGH5) [reply rsts/concepts/basics.rst:102]: Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.

File: rsts/concepts/basics.rst (hunk 1 above)
Comment (SandraGH5) [reply rsts/concepts/basics.rst:17]: Thanks Niels. Will run the editorial review after earlier conflicts have been resolved.

**Review by EngHabu — APPROVED** [review]

**Pull request merged**